app/controllers/user_menus_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UserMenusController < Api::V2::BaseController
```

Comment:
I think this should be inheriting `ApplicationController`?

Reply:
I've noticed in the `/status` endpoint that if you request it with `Accept: application/html` (as a browser does) it will return some odd data. It's ok if you don't specify it or explicitly request JSON but an API controller probably deals better with this.

Reply:
API controller is a better fit because it responds always with JSON, that way we get error handling for free. `ApplicationController` responds with HTML when user is not logged in.

Reply:
also similar to the notifications controller.